.editorconfig

```diff
@@ -106,6 +106,11 @@ dotnet_style_predefined_type_for_member_access = true:suggestion
 #prefer the language keyword for local variables, method parameters, and class members, instead of the type name, for types that have a keyword to represent them
 dotnet_style_predefined_type_for_locals_parameters_members = true:suggestion
 
+#show an IDE warning when default access modifiers are explicitly specified - the first line;
+#also make sure to raise a warning on build (not just an IDE warning ... because apparently there's a difference) - the second line
```

Comment:
Why don't you put the comments above the lines respectively instead of writing "the first line" and "the second line"?

Reply:
Perhaps you're right.